src/esp/bindings/SimBindings.cpp

```diff
@@ -52,6 +52,11 @@ void initSimBindings(py::module& m) {
           R"(See tutorials/async_rendering.py)")
       .def_readwrite("frustum_culling", &SimulatorConfiguration::frustumCulling)
       .def_readwrite("enable_physics", &SimulatorConfiguration::enablePhysics)
+      .def_readwrite(
+          "use_semantic_textures",
+          &SimulatorConfiguration::useSemanticTexturesIfFound,
+          R"(If the loaded scene/dataset supports semantically annotated textures, use these for
+      semantic rendering.)")
```

Comment:
Same as in the viewer here, I think the default would make more sense to be inverted.